core/iwasm/aot/aot_runtime.c

```diff
@@ -2880,7 +2937,7 @@ aot_invoke_native(WASMExecEnv *exec_env, uint32 func_idx, uint32 argc,
             goto fail;
         }
 #if WASM_ENABLE_AOT_STACK_FRAME != 0
-        struct WASMInterpFrame *prev_frame = exec_env->cur_frame;
+        void *prev_frame = get_top_frame(exec_env);
 
         if (!aot_alloc_frame(exec_env, func_idx)) {
```

Comment:
Should check `!is_frame_per_function(exec_env)`?

Reply:
no, we do want to allocate frame in the `call` instruction for the imported and/or native functions even in the `frame-per-function` mode, otherwise this function will not be included in the callstack (see my comment https://github.com/bytecodealliance/wasm-micro-runtime/pull/3773#discussion_r1746724221)